core/src/Exceptions/CoreErrorCodes.cs

```diff
@@ -54,6 +54,7 @@ internal class CoreErrorCodes
         public const string InvalidAuthority = "invalid_authority";
 
         public const string PlatformNotSupported = "platform_not_supported";
+        public const string PlatformNotInitialized = "platform_not_initialized";
```

Comment:
>PlatformNotInitialized [](start = 28, length = 22)

Why not the one above? (same for the messages). I believe it was made for the same reason (and most likely this change is removing the use of the one above)

Reply:
I believe this should actually be InternalError - if the users sees this they should log a bug.

Reply:
sounds good.

---
In reply to: [215757011](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1244#discussion_r215757011) [](ancestors = 215757011)